iati-activities-schema.xsd

```diff
@@ -2804,5 +2806,13 @@
     </xsd:annotation>
   </xsd:attribute>
 
+  <xsd:attribute name="vocabulary" type="xsd:string">
```

Comment:
`vocabulary` is a commonly-used term, so would suggest naming this `aidTypeVocabulary`.